### DISCUSSION TITLE
170

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#132] [General]
# 潮流周刊第170期 - 杭州翠绿

封面图来源于周末小区随手拍，杭州的春天是真的很好看，就是时间感觉不是很长，前 1 个月还是冬天，下一个月就是夏天短袖了，这段时间的温度还是可以接受的，需要多出去走走看看，防止后面到 5 月后的暴热。

https://weekly.tw93.fun/posts/170-%E6%9D%AD%E5%B7%9E%E7%BF%A0%E7%BB%BF

**Comment by tsuzurux** [comment]
3.31号晚偷发？嘿嘿

**Reply by tw93** [reply]
哈哈 毕竟是打工人

**Comment by mumingfang** [comment]
投稿一个浏览器插件，橘猫起始页，官网 https://jvmao.net/， 介绍贴：https://www.v2ex.com/t/1028236

**Comment by duofu** [comment]
“是由一个伟大的想法、一个伟大的团队、一个伟大的产品....”看到这几个字眼（伟大XX，伟大XX）就反胃，妥妥的成功学，屁用没有

**Reply by tw93** [reply]
其实吧 应该是老美英文到中文的翻译问题导致，我觉得叫 牛逼 更好，不过我更建议是看完内容再评价，而非看到概要来评价
